mypy/semanal.py

```diff
@@ -4156,3 +4189,8 @@ def visit_any(self, t: AnyType) -> Type:
         if t.type_of_any == TypeOfAny.explicit:
             return t.copy_modified(TypeOfAny.special_form)
         return t
+
+
+def get_prefix(fullname: str) -> str:
```

Comment:
This should really be moved to util so you won't need three copies of the same function.